lib/mixins/mutable-data.html

```diff
@@ -105,7 +105,7 @@
 
     }
     /** @type {boolean} */
-    MutableData.prototype.mutableData = false;
+    MutableData.prototype.mutableData = true;
```

Comment:
So, `this.mutableData` is never used in `MutableData`, only `OptionalMutableData` (`MutableData` doesn't have a switch and always calls `mutablePropertyChange` with its `mutableData` arg set to `true`).

So it looks like this line is altogether unnecessary.  The blame shows it being added to help closure understand property types, but it must have been made in error; I don't see the closure warning count change my removing it, so I'd suggest just updating the PR to remove it altogether.